src/test/session-details-provider/README.md

```diff
@@ -7,7 +7,10 @@ without those tests needing to actually receive the code via text or email.
 
 `session-details-manifest.yml` is used to deploy it to cloud foundry, by running this command from the root of the project:
 ```
-cf push -f src/test/session-details-provider/session-details-manifest.yml --var session_details_app_name=htbhf-session-details-staging --var session_secret=ReplaceWithTheRealSessionSecret
+cf push -f src/test/session-details-provider/session-details-manifest.yml \
```

Comment:
Unrelated change that I forgot was in my workspace. I added the missing `session_details_host_name` variable to the example push command. Let me know if it needs to come out...